sdk/mgmtcommon/TestFramework/ClientRuntime.Azure.TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework.csproj

```diff
@@ -58,21 +58,21 @@
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' ">
     <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="[3.14.0, 4.0.0)" />
-    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="[5.1.2]" />
```

Comment:
Their are issues in the past that occurred which led us to pin ADAL dependencies to a particular versions.
I do not see the reason mentioned in the PR for adding a range.
Plus be very careful to update any ADAL dependencies, you need to have a very valid reason to update any ADAL dependencies.
This test framework is used not only for .NET SDKs but for also powershell and Fluent SDK
So make sure all those components are working as expected with this change.

Reply:
The reason for it stems from the fact that the framework dependency enforces this version on the SDK. In all frankness there is no reason why this would matter for the sdk's purposes, however, Microsoft.AspNetCore.App 2.2.0 appears to require version 5.3 of this package and above, in other words the generated nuget package is incompatible with a brand new web app with nothing on it  which likely means the Sdk will face significant incompatibility. Dozie mentioned this was a possible way forward. I don't have much experience with nuget dependencies and it does feel like there should be a better way (the dependency enforcements of the test framework should have nothing to do with those used by the sdk in my likely misinformed opinion) . Any help would be appreciated.

Reply:
@estebanreyl I will remove the changes from this PR, while I continue to investigate how to resolve your issue.

Reply:
@chidozieononiwu I greatly appreciate the help